compass/Coordinator.java

```diff
@@ -178,10 +178,18 @@ private boolean nodeMatchesInternalState(GetNodeInfoResponse nodeInfo) {
     return config.inception || (nodeInfo.getLatestSolidSubtangleMilestoneIndex() == state.latestMilestoneIndex);
   }
 
-  private void broadcastLatestMilestone() throws ArgumentException {
+  private void broadcastLatestMilestone() throws InterruptedException {
     if (config.broadcast) {
       for (String tx : state.latestMilestoneTransactions) {
-        api.storeAndBroadcast(tx);
+        for(int i = 0; i < config.APIRetries; i++) {
+          try {
+            api.storeAndBroadcast(tx);
```

Comment:
We should make sure how all these API calls can fail. Is `ArgumentException` if the call returns a 500 or times out?